### DISCUSSION TITLE
new update caused error can no longer open iw3

### DISCUSSION THREAD

**Discussion by kaelsonofkrypto** [nagadomi/nunif#190] [General]
hi the new update caused an error and i cant open iw3 anymore after the update please see screenshot below

**Comment by nagadomi** [comment]
see https://github.com/nagadomi/nunif/issues/189
I already fixed this error, run `update.bat` again.

**Reply by kaelsonofkrypto** [reply]
ok i will try again

**Reply by kaelsonofkrypto** [reply]
> see #189 I already fixed this error, run `update.bat` again.

thanks, its working again
